filters/color-overlay/src/ColorOverlayFilter.ts

```diff
@@ -27,15 +27,19 @@ type Color = number | number[] | Float32Array;
 class ColorOverlayFilter extends Filter
 {
     private _color = 0x0;
+    private _alpha = 0;
 
     /**
      * @param {number|Array<number>} [color=0x000000] - The resulting color, as a 3 component RGB e.g. [1.0, 0.5, 1.0]
+     * @param {number} [alpha=0] - The alpha value of the color
```

Comment:
The default should be `1`, no? I'd assuming setting `0` would make the color overlay invisible. To not be breaking change, this should be fully visible.

Reply:
set it to `1`